dandiapi/settings.py

```diff
@@ -83,3 +83,8 @@ class ProductionConfiguration(DandiMixin, ProductionBaseConfiguration):
 class HerokuProductionConfiguration(DandiMixin, HerokuProductionBaseConfiguration):
     # All login attempts in production should go straight to GitHub
     LOGIN_URL = '/accounts/github/login/'
+
+
+# use the same config for staging as production, but also allow regex for OAuth redirect URIs
+class HerokuStagingConfiguration(HerokuProductionBaseConfiguration):
```

Comment:
The heroku config variable `DJANGO_CONFIGURATION` would need to be changed to `HerokuStagingConfiguration` to support this.